content/academy/pro_apify_scraper_development/managing_source_code.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Managing source code
+description: Learn how to manage your actor's source code more efficiently by integrating it with a Github repository. This is the standard on the Apify platform.
+menuWeight: 6.3
+paths:
+    - pro-apify-scraper-development/managing-source-code
+---
+
+# [](#managing-source-code) Managing source code
+
+In this brief lesson, we'll discuss how to better manage an actor's source code. Up 'til now, you've been developing your scripts locally, then pushing the code directly to the actor on the Apify platform; however, there is a much more optimal _(and standard)_ way.
+
+## [](#learning) Learning 🧠
+
+Thus far, every time we've updated our code on the Apify platform, we've used the `apify push` CLI command; however, this can be problematic for a few reasons - mainly because, if someone else wants to make a change to/maintain your code, they don't have access to it, as it is on your local machine.
+
+If you're not yet familiar with Git, please get familiar with it through the [Git documentation](https://git-scm.com/docs), then take a quick moment to read about [Github integration](https://docs.apify.com/actors/development/source-code#github-integration) in the Apify docs.
+
+Also, try to explore the **Multifile editor** in one of the actors you developed in the previous lessons before moving forward.
+
+## [](#quiz) Knowledge check 📝
+
+1. Do you have to rebuild an actor each time the source code is changed?
+2. In git, what is the difference between **pushing** changes and making a **pull request**?
+3. Based on your knowledge and experience, is the `apify push` command worth using (in your opinion)?
+
+## [](#our-task) Our task
+
+> This lesson's task is so quick and easy, we won't even be splitting this topic into two parts like the previous two topics!
+
+First, we must initialize a Github repository (you can use Gitlab if you'd like, but this lesson's examples will be using Github). Then, after pushing our main Amazon actor's code to the repo, we must switch our actor's source code to use the content of the Github repository instead.
+
+## [](#integrating-github) Integrating Github source code
+
+First, let's create a repository. This can be done [in a number of ways](https://kbroman.org/github_tutorial/pages/init.html), but the way we'll do it is by creating the remote repository on Github's website:
```

Comment:
```suggestion
First, let's create a repository. This can be done [in a number of ways](https://kbroman.org/github_tutorial/pages/init.html), but we'll do it by creating the remote repository on Github's website:
```